docs/source/contributing/developing.adoc

```diff
@@ -370,6 +371,30 @@ This information is often enough to find and debug the reason for a failure.
 However, for cases which require further investigation, the integration test also logs more detailed progress into a log file.
 This file is located at *_$GOPATH/github.com/minishift/minishift/out/test-results/integration_YYYY-MM-DD_HH:MM:SS.log_*.
 
+[[writing-features]]
+==== Writing features
+
+Minishift integration testsuite offers comprehensive set of implemented Gherkin steps which allow for comfortable writing of feature files.
+All available steps are defined in file link:https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go[/test/integration/testsuite.go].
+Definitions of steps are simple and self-documenting, for example:
+
+----
+s.Step(`^executing "minishift (.*)" (succeeds|fails)$`,
+    ExecutingMinishiftCommandSucceedsOrFails)
+----
+
+defines a step which matches regular expression `^executing "minishift (.*)" (succeeds|fails)$`. If matched the two capturing groups are passed as parameters to the function `ExecutingMinishiftCommandSucceedsOrFails(command string, expectedResult string)`, which implements the actual behaviour of the step.
```

Comment:
"This defines a step which matches the regular expression"

"If matched, the two" (Add a comma.)

Also, please add a newline between sentences here. :)